### PR TITLE
docs: AD Introduction

### PR DESCRIPTION
**Pull request by justin-stephenson**

### CODE REVIEW

File: src/docs/ad/ad-introduction.rst [hunk 1]
```diff
@@ -1,2 +1,40 @@
 Introduction to Active Directory Integration
-############################################
\ No newline at end of file
+############################################
+
+Direct Active Directory integration
+-----------------------------------
+
+Heterogeneous IT environments often contain various domains and operating systems that need to be able to seamlessly communicate. SSSD offers integration with Active Directory on Linux/Unix clients by taking advantage of the SSSD AD provider. In addition to retrieving and caching user and group information from Active Directory, SSSD can:
```
Comment (pbrezina) [inline src/docs/ad/ad-introduction.rst:7]: In general, SSSD runs on Unix systems to some level but we, developers, have nothing to do with Unix. So perhaps it is better to omit it so we don't advertise a false assumption that we run tests against Unix systems?

File: src/docs/ad/ad-introduction.rst [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+* AD Site discovery
+* Automatically renew linux host computer object
+
+To read more about how SSSD is used in AD integration at a high level, refer to the following links:
+
+`Understanding SSSD and its benefits <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_authentication_and_authorization_in_rhel/understanding-sssd-and-its-benefits_configuring-authentication-and-authorization-in-rhel>`_
+
+
+`Why Use SSSD Instead of a Direct LDAP Configuration for Applications?  <https://www.redhat.com/en/blog/why-use-sssd-instead-direct-ldap-configuration-applications>`_
+
```
Comment (pbrezina) [inline src/docs/ad/ad-introduction.rst:26]: Make it a list?

* link1
* link2

File: src/docs/ad/ad-introduction.rst [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+Or on the terminal to read about SSSD's AD provider
+
+.. code-block:: bash
```
Comment (pbrezina) [inline src/docs/ad/ad-introduction.rst:29]: console

File: src/docs/ad/ad-introduction.rst [hunk 4]
```diff
@@ -1,2 +1,40 @@
...
+
+    $ man sssd-ad
+
+Active Directory integration through FreeIPA
+--------------------------------------------
+
+SSSD can also retrieve information and perform authentication against Active Directory Domain Controllers through IPA servers, this is done via IPA - AD Trust - also called ``Indirect AD`` integration. Red Hat documentation gives more information about how this works.
```
Comment (pbrezina) [inline src/docs/ad/ad-introduction.rst:36]: Italics instead of code? `*Indirect AD*`

File: src/docs/ad/ad-introduction.rst [hunk 5]
```diff
@@ -1,2 +1,40 @@
...
+
+`Planning a cross-forest trust between IdM and AD and Installing a trust between IdM and AD. <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/planning_identity_management/index?lb_target=production#planning-a-cross-forest-trust-between-idm-and-ad_planning-dns-and-host-names>`_
+
+`Integrating IdM and AD. <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/installing_identity_management/index?lb_target=production#installing-trust-between-idm-and-ad_installing-identity-management>`_
```
Comment (pbrezina) [inline src/docs/ad/ad-introduction.rst:40]: Make it a list?

**Review by pbrezina — CHANGES_REQUESTED** [review]
Hi Justin, minor comments are inside.

**Comment by pbrezina** [comment]
Also please double  check that the links to Red Hat articles are free to access outside VPN and authorized access.

**Comment by justin-stephenson** [comment]
All changes made as requested. Links are external.

**Comment by pbrezina** [comment]
Thank you. Ack.

**Pull request merged**